frontend/projects/shared/styles/shared.scss

```diff
@@ -110,4 +110,12 @@ ion-modal::part(content) {
 
 .montserrat {
   font-family: 'Montserrat', sans-serif !important;
+}
+
+.color-success-shade {
```

Comment:
I _think_ you can also do this inline with style="color: var(--ion-color-success-shade)" pretty sure.

Reply:
You can. I abstracted it as I prefer to not use inline styles, for future usability, and for readability.